### PR TITLE
Move ci to GitHub actions

### PR DESCRIPTION
**Pull request by sjagoe**
Add CI with github actions for Windows, MacOS and Linux

### CODE REVIEW

**Comment by sjagoe** [comment]
The mac os tests need some attention so that we can have them legitimately succeed.

**Comment by sjagoe** [comment]
And I'll update the PR later to remove travis and appveyor

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+          - os: windows-2019
+            python: "3.5"
+            toxenv: cp35
+            continue-on-error: false
+
+          # 3.10 Alpha
+          - os: ubuntu-18.04
+            python: "3.10.0-alpha.2"
+            toxenv: "cp310"
+            continue-on-error: false
```
Comment (itziakos) [inline .github/workflows/test.yml:97]: Should we set this also to true?

**Review by itziakos — APPROVED** [review]
Just one question. PR looks ok

File: .github/workflows/test.yml (hunk 1 above)
Comment (sjagoe) [reply .github/workflows/test.yml:97]: We can do that if it starts failing in a newer alpha.

**Pull request merged**